webview-ui/src/components/ResolutionsPage.tsx

```diff
@@ -131,8 +131,8 @@ const ResolutionPage: React.FC = () => {
               </CardTitle>
               <CardBody>
                 <IncidentList
-                  incidents={solutionScope?.incident ? [solutionScope?.incident] : []}
-                  selectedIncident={solutionScope?.incident}
+                  incidents={solutionScope?.incidents ?? []}
+                  selectedIncident={solutionScope?.incidents?.[0]}
```

Comment:
Looks like this will need to updated once multiple incidents are actually requested.  The resolution view needs to be able to handling showing a set of incidents that have been requested to be fixed.

Probably need an issue to track this.

Reply:
It seems that without this change we won't be able to view the resolutions so I will fix it as part of the #171